server/implementation/src/main/java/io/smallrye/graphql/bootstrap/Bootstrap.java

```diff
@@ -22,6 +22,7 @@
 import javax.json.JsonReaderFactory;
 import javax.json.bind.Jsonb;
 
+import graphql.com.google.common.collect.Streams;
```

Comment:
Is this the Streams class you intended to use ??

Reply:
Whoa, good catch, I didn't even realize I was using something not from the JDK api. The IDE suggested it and it just did what I needed. Ok, I just rewrote it to use plain Java streams.